src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1640,6 +1643,7 @@
   </Node>
 
   <Node Name="BoundCollectionInitializerExpression" Base="BoundObjectInitializerExpressionBase">
+    <Field Name="PlaceholderOpt" Type="BoundLValuePlaceholder" Null="allow" />
```

Comment:
>BoundLValuePlaceholder [](start = 39, length = 22)

The name feels somewhat confusing. Usually, LValue is used to refer to something that can be assigned to, a local, a field, an array element, etc.

Reply:
This comment is for name of the type ```BoundLValuePlaceholder```

---
In reply to: [325362467](https://github.com/dotnet/roslyn/pull/38732#discussion_r325362467) [](ancestors = 325362467)